docs/UserGuide.md

```diff
@@ -394,12 +394,15 @@ Parameters:
 in the filtered contact list must have part of the `[term]` in the `[field]`
 specified above.
 * `[taken]` (optional). Only for filtering by `course`, this parameter additionally filters
-  out contacts who have finished the course.
+  out contacts who have finished/haven't started the course at the current date of your system.
```

Comment:
"as of" seems like the better phrase. Are you ok to change this?

Reply:
Noted, and fixed 👍